src/Core/Utilities/StaticStore.cs

```diff
@@ -106,382 +106,11 @@ static StaticStore()
-                Product = ProductType.Families,
-                Name = "Families 2019",
-                IsAnnual = true,
-                NameLocalizationKey = "planNameFamilies",
-                DescriptionLocalizationKey = "planDescFamilies",
-                BaseSeats = 5,
-                BaseStorageGb = 1,
-                MaxUsers = 5,
-
-                HasAdditionalStorageOption = true,
-                HasPremiumAccessOption = true,
-                TrialPeriodDays = 7,
-
-                HasSelfHost = true,
-                HasTotp = true,
-
-                UpgradeSortOrder = 1,
-                DisplaySortOrder = 1,
-                LegacyYear = 2020,
-
-                StripePlanId = "personal-org-annually",
-                StripeStoragePlanId = "storage-gb-annually",
-                StripePremiumAccessPlanId = "personal-org-premium-access-annually",
-                BasePrice = 12,
-                AdditionalStoragePricePerGb = 4,
-                PremiumAccessOptionPrice = 40,
-
-                AllowSeatAutoscale = false,
-            },
-            new Plan
-            {
-                Type = PlanType.TeamsAnnually2019,
-                Product = ProductType.Teams,
-                Name = "Teams (Annually) 2019",
-                IsAnnual = true,
-                NameLocalizationKey = "planNameTeams",
-                DescriptionLocalizationKey = "planDescTeams",
-                CanBeUsedByBusiness = true,
-                BaseSeats = 5,
-                BaseStorageGb = 1,
-
-                HasAdditionalSeatsOption = true,
-                HasAdditionalStorageOption = true,
-                TrialPeriodDays = 7,
-
-                HasTotp = true,
-
-                UpgradeSortOrder = 2,
-                DisplaySortOrder = 2,
-                LegacyYear = 2020,
-
-                StripePlanId = "teams-org-annually",
-                StripeSeatPlanId = "teams-org-seat-annually",
-                StripeStoragePlanId = "storage-gb-annually",
-                BasePrice = 60,
-                SeatPrice = 24,
-                AdditionalStoragePricePerGb = 4,
-
-                AllowSeatAutoscale = true,
-            },
-            new Plan
-            {
-                Type = PlanType.TeamsMonthly2019,
-                Product = ProductType.Teams,
-                Name = "Teams (Monthly) 2019",
-                NameLocalizationKey = "planNameTeams",
-                DescriptionLocalizationKey = "planDescTeams",
-                CanBeUsedByBusiness = true,
-                BaseSeats = 5,
-                BaseStorageGb = 1,
+        var passwordManagerPlans = PasswordManagerPlanStore.CreatePlan();
+        var secretManagerPlans = SecretsManagerPlanStore.CreatePlan();
 
-                HasAdditionalSeatsOption = true,
-                HasAdditionalStorageOption = true,
-                TrialPeriodDays = 7,
+        Plans = passwordManagerPlans.Concat(secretManagerPlans);
```

Comment:
Wouldn't it be better to have `passwordManagerPlans` separate from  `secretManagerPlans`? Is there a need to have all plan types in the same collection?